finalPrep/modZeroHero.js

```diff
@@ -55,18 +80,49 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Beetlejuice",
+  smell: "rotten garbage",
+  weight: 211,
+  citiesDestroyed: ['Winter River', 'myTown', 'yourTown'],
+  luckyNumbers: [13, 666],
+  address: bjAddress = {
```

Comment:
You can assign the key `address` directly to the object (no need to declare a variable name).